gravitee-gateway-standalone/gravitee-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -119,6 +119,10 @@ reporters:
   # logging configuration
   #logging:
   #  max_size: -1 # max size per API log content respectively : client-request, client-response, proxy-request and proxy-response in MB (-1 means no limit)
+  #  ssl:
```

Comment:
Please add some comments to explain the purpose of the SSL keys.

Reply:
ok